selftests/functional/test_plugin_jobscripts.py

```diff
@@ -42,6 +42,19 @@
 warn_non_zero_status = True"""
 
 
+TEST_RESULT_FILES = """#!/bin/sh -e
+# Post script should run before results archivation
+test -f ${AVOCADO_JOB_LOGDIR}.zip && exit 1
+# But results.json should exist
```

Comment:
and `fobar.baz` in the future... IMO the `results.json` as an example in comment inside selftest is sufficient